webapp/channels/src/components/suggestion/suggestion_list.tsx

```diff
@@ -38,12 +34,11 @@ export interface Props {
 
 export default class SuggestionList extends React.PureComponent<Props> {
     static defaultProps = {
-        renderDividers: [],
         renderNoResults: false,
     };
-    contentRef: React.RefObject<HTMLUListElement>;
+    contentRef: React.RefObject<HTMLElement>;
     wrapperRef: React.RefObject<HTMLDivElement>;
-    itemRefs: Map<string, HTMLElement>;
+    itemRefs: Map<string, HTMLElement | null>;
```

Comment:
Should this just be `React.ReactNode`?

Reply:
No, it can't. `React.ReactNode`s are what's sent to React to be put on the virtual DOM, but this holds references to the actual DOM nodes (or null after those nodes are unmounted)